bundles/test/org.eclipse.smarthome.magic/src/main/java/org/eclipse/smarthome/magic/binding/handler/MagicColorLightHandler.java

```diff
@@ -34,8 +32,6 @@ public MagicColorLightHandler(Thing thing) {
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
-        if (channelUID.getId().equals(CHANNEL_COLOR)) {
-        }
```

Comment:
Since the magic bundle is supposed to be a "demo bundle" to play around with and show how to implement certain things I would vote for adding a comment inside the empty if branch instead of removing it. I.e. something like `//process the command for the color channel here`

Reply:
Okay, but even if there is a comment in the empty if branch, it will still be reported by the SAT.

Reply:
Good point. How about adding that comment just before the branch instead and in addition commenting out the whole branch? This way developers will see that someone took care of this condition and the SAT tool will not complain because that branch doesn't exist anymore in code, but in a comment?